pkgs/tools/misc/hdf5/default.nix

```diff
@@ -45,6 +49,7 @@ stdenv.mkDerivation rec {
   configureFlags = []
     ++ optional cpp "--enable-cxx"
     ++ optional (gfortran != null) "--enable-fortran"
+    ++ optional fortran2003 "--enable-fortran2003"
```

Comment:
Why not just throw ```--enable-fortran2003``` in with ```--enable-fortran```?

Reply:
Users might want to use the Fortran90 bindings instead of the Fortran2003 ones.